### PR TITLE
Fix remotes

### PR DESCRIPTION
**Pull request by spladug**
It's useful to have the ability to run a very simple thrift client from the CLI when devving a service. Thrift comes with tooling for this built in, but it was broken on Python 3. When originally packaging this I tried to fix it on Python 3 and broke it on Py2 in the process. This makes it so we can use the remote tool from both versions of the language. There's a bit more churn in this diff than reality, just because of splitting up stuff properly and what that means to quilt. Here's the operative diff from before to after in the actual output packaged code:

```diff
diff --git a/thrift/lib/py/util/randomizer.py b/thrift/lib/py/util/randomizer.py
index 02f1afc..cd10671 100644
--- a/thrift/lib/py/util/randomizer.py
+++ b/thrift/lib/py/util/randomizer.py
@@ -387,7 +387,10 @@ def _integer_randomizer_factory(name, ttype, n_bits):
             else:
                 raise TypeError("Invalid %s seed: %s" % (_name, seed))
 
-    NBitIntegerRandomizer.__name__ = "%sRandomizer" % _name
+    if six.PY2:
+        NBitIntegerRandomizer.__name__ = str("%sRandomizer" % _name)
+    else:
+        NBitIntegerRandomizer.__name__ = "%sRandomizer" % _name
 
     return NBitIntegerRandomizer

```

tl;dr is that `from __future__ import unicode_literals` is on, so in both Py2 and Py3 `"blah"` is unicode. Py3 is cool with unicode being assigned to a class's `__name__` (and in fact requires that it not be `bytes`) but Py2 is not.

I also took the opportunity to fix some issues in the README.

Once this is done, I'll add docs to the cookiecutter on how to use the remote. It looks something like this:

```
ubuntu@activity:~/src/activity$ python -m activity.activity_thrift.remote
Usage: remote.py [OPTIONS] FUNCTION [ARGS ...]

Options:

-c --compact         Use TCompactProtocol
-f --framed          Use framed transport
-h HOST[:PORT], --host HOST[:PORT] The host and port to connect to
-j --json            Use TJSONProtocol
-s --ssl             Use SSL socket
-U --unframed        Use unframed transport
-F --fuzz            Use TFuzzyHeaderTransport to send a fuzzed message for testing thrift transport. Optionally include a list of message field names to fuzz after this flag. Fields: magic, seq_id, proto_id, num_transforms, padding, length, version, flags, header_size, transform_id, padding_bytes, payload, identity
-u --url             The URL to connect to, for HTTP transport

ubuntu@activity:~/src/activity$ python -m activity.activity_thrift.remote -h localhost:9090 count_activity some_context
ActivityInfo(
    count=0,
    is_fuzzed=True)
```

### CODE REVIEW

**Review by dellis23 — CHANGES_REQUESTED** [review]
Awesome!  Thanks for that.

**Comment by dellis23** [comment]
Err.  Gotta get used to this github review thing.

**Comment by dellis23** [comment]
I don't see an obvious way to undo my "changes requested," so consider this PR approved.

**Review by ckwang8128 — APPROVED** [review]
